build/build-runtimeimages-bases.sh

```diff
@@ -85,7 +85,7 @@ for dockerFile in $dockerFiles; do
     cd $REPO_DIR
 
     echo
-    docker build -f $dockerFile -t $localImageTagName $labels .
+    docker build --no-cache -f $dockerFile -t $localImageTagName $labels .
```

Comment:
As we discussed in our meeting, should we have this behind a flag so that we do it only when required?

Reply:
I will like that to be adhoc

Reply:
I've changed the approach now, so only the steps we want to always run won't be cached.